assets/src/js/enhancedDonateButton.js

```diff
@@ -1,34 +1,58 @@
-export const setupEnhancedDonateButton = function() {
-  function isMobile() {
-    return window.matchMedia('(max-width: 576px)').matches;
-  }
+export const setupEnhancedDonateButton = () => {
+  const isMobile = () => window.matchMedia('(max-width: 576px)').matches;
 
-  function setupDonateButton() {
+  const setupDonateButton = () => {
     if ( isMobile() ) {
-      if (!$('.btn-enhanced-donate.btn-donate-top').length) {
-        $('.btn-enhanced-donate').clone().appendTo('body').addClass('btn-donate-top');
-        $('.btn-donate-top').parent('body').addClass('with-donate-on-top');
+      if ( ! document.querySelector('.btn-enhanced-donate.btn-donate-top') ) {
+        const enhancedDonateBtn = document.querySelector('.btn-enhanced-donate').cloneNode(true);
+        enhancedDonateBtn.classList.add('btn-donate-top');
+
+        document.querySelector('body').appendChild(enhancedDonateBtn);
+        document.querySelector('body').classList.add('with-donate-on-top');
+      }
+    } else {
+      const buttonClass = document.querySelector('.btn-donate-top');
```

Comment:
I would rename this variable (maybe to just `button`?) since the selector returns the whole element and not just the class 🙂